getting_started/workflow/project_setup/version_control_system.rst

```diff
@@ -0,0 +1,31 @@
+.. _doc_version_contro_system:
+
+Version Control System
+======================
+
+Introduction
+------------
+
+Godot supports the use of version control systems, they can be
+setup or shut down in the editor under Project, Version Control.
+
+.. image:: img/version_control_menu.png
+
+However, version control does not work out of the box and requires
+a plugin for the specific vcs you are using.
+
+Official Git plugin
+-------------------
+
+Git is supported with an official plugin, you can find the latest
+releases `here <https://github.com/godotengine/godot-git-plugin/releases>`_ 
+(it is not available in the asset library). Documentation on how
+to use the git plugin can be found `here <https://github.com/godotengine/godot-git-plugin/wiki>`_.
+
+Files to exclude from VCS
+-------------------------
+
+There are some folders Godot creates you should have your VCS ignore.
```

Comment:
I suggest putting a colon here and formatting the list of folders with `::`, like we do with code.